stock_move_backdating/stock.py

```diff
@@ -81,10 +81,10 @@ def on_change_date_backdating(self, cr, uid, ids, date_backdating,
         if (dt > NOW):
             warning = {'title': _('Error!'), 'message': _(
                 'You can not process an actual movement date in the future.')}
-            values = {
-                'date_backdating': NOW.strftime(DEFAULT_SERVER_DATETIME_FORMAT)
-            }

```

Comment:
I understand but the `value` is setting the date that is in the future back to the `today` date to prevent setting date in the future.
After your change, it might be possible to set a date in the future as the warning isn't preventing changes. It's just a warning.